### ISSUE TITLE
not sending sms after press send

### ISSUE THREAD

**Issue by GoogleCodeExporter** [satyajit319/websmsdroid#387]
```
What steps will reproduce the problem?
1. Wditing sms witn normal and less than 160 characters, and pressing send
2.
3.

What is the expected output? What do you see instead?
If I press send text area gets empty, but no sms is send

What version of the product are you using? On what operating system?
websms v3.3, android 2.2

Please provide any additional information below.
After update yesterday I send sms, but from now sms don't get send. I also 
tried to restart, without change.
```

Original issue reported on code.google.com by `brianhu...@gmail.com` on 19 Nov 2010 at 12:08

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
I have the same problem with WebSMS + FishText.

Problem started after update.

websms v3.3, android 2.2

```

Original comment by `backup.t...@gmail.com` on 19 Nov 2010 at 12:20

**Comment by GoogleCodeExporter** [comment]
```
please send me your logs via websms preferences after sending a message.
```

Original comment by `felix.bechstein` on 19 Nov 2010 at 3:19
- Changed title: **not sending sms after press send**
- Changed state: **Accepted**
- Added labels: **Component-Logic**, **Priority-Critical**, **Product-WebSMS**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
please verify this fix
```

Original comment by `felix.bechstein` on 19 Nov 2010 at 4:44
- Changed state: **ShouldBeFixed**

Attachments:
- [WebSMS-release.apk](https://storage.googleapis.com/google-code-attachments/websmsdroid/issue-387/comment-4/WebSMS-release.apk)

**Comment by GoogleCodeExporter** [comment]
```
I've send the log. The fix works! The thing is that the official released v3.3 
also works right after installing (I tested this twice), somehow it stopped 
working after a while and gave the problem mentioned above.
I hope the fix keeps working, if not I'll write a comment here.

Thanks a lot for your help! Vielen dank! :)
```

Original comment by `backup.t...@gmail.com` on 19 Nov 2010 at 6:16

**Comment by GoogleCodeExporter** [comment]
```
the ' sign was the problem..
```

Original comment by `felix.bechstein` on 19 Nov 2010 at 6:30
- Changed state: **Fixed**
